lib/shelley/test/integration/Main.hs

```diff
@@ -168,21 +170,21 @@ main = withUtf8Encoding $ withTracers $ \tracers -> do
                     ByronTransactions.spec @n
                     ByronHWWallets.spec @n
 
-                -- possible conflict with StakePools
-                Settings.spec @n
+                    -- possible conflict with StakePools
+                    Settings.spec @n
 
-            -- Hydra runs tests with code coverage enabled. CLI tests run
-            -- multiple processes. These processes can try to write to the
-            -- same .tix file simultaneously, causing errors.
-            --
-            -- Because of this, don't run the CLI tests in parallel in hydra.
-            parallelIf (not nix) $ describe "CLI Specifications" $ do
-                AddressesCLI.spec @n
-                TransactionsCLI.spec @n
-                WalletsCLI.spec @n
-                HWWalletsCLI.spec @n
-                PortCLI.spec @t
-                NetworkCLI.spec @t
+                -- Hydra runs tests with code coverage enabled. CLI tests run
+                -- multiple processes. These processes can try to write to the
+                -- same .tix file simultaneously, causing errors.
+                --
+                -- Because of this, don't run the CLI tests in parallel in hydra.
+                parallelIf (not nix) $ describe "CLI Specifications" $ do
```

Comment:
The "CLI Specification" should not be inside "API Specifications". 

The "Settings.spec @n" indentation looks good though.